.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,10 +18,13 @@
 - [ ] My name is on the [list of contributors](https://github.com/glotzerlab/signac/blob/master/contributors.yaml).
 - [ ] My code follows the [code style guideline](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md#code-style) of this project.
 - [ ] The changes introduced by this pull request are covered by existing or newly introduced tests.
-
-If necessary:
-- [ ] I have updated the API documentation as part of the package doc-strings.
-- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
+- [ ] I have determined whether the [framework documentation](https://docs.signac.io/) requires an update (see guidance below). If so, I have created a separate pull request to update it on [signac-docs](https://github.com/glotzerlab/signac-docs) at this link: glotzerlab/signac-docs/pull#**XXX**
+<!-- The framework documentation must be updated if:
+        This PR will change user-facing features (labeled "enhancement").
+        This PR will break things (labeled "breaking").
+-->
+<!-- The framework documentation can be left if:
+        This PR affects only internal components.
+-->
 - [ ] I have updated the [changelog](https://github.com/glotzerlab/signac/blob/master/changelog.txt) and added all related issue and pull request numbers for future reference (if applicable). See example below.
```

Comment:
Similar issue here with “example below” which is hidden. 🙃

Reply:
I'll just get rid of it because people can refer to the changelog file itself for examples